src/reverse.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+import {sizeOfShape, Tensor} from './lib/tensor.js';
+import {validateReduceParams} from './lib/validate-input.js';
+
+/**
+ * Reverse input along the given axes.
+ * @param {Tensor} input
+ * @param {MLReverseOptions} [options]
+ * @return {Tensor}
+ */
+export function reverse(input, {axes}) {
+  // reuse validateReduceParams to check parameters of reverse
+  validateReduceParams(input, {axes});
```

Comment:
If `reverse` had been before `reduce`, would `reduce()` call `validateReverseParams`? Renaming `validateReduceParams` to a more generic `validateAxes` would make sense.